.github/workflows/ci.yml

```diff
@@ -53,7 +52,7 @@ jobs:
           compiler: ${{ matrix.dc }}
       - name: Install multi-lib for 32-bit systems
         if: matrix.arch == 'x86' && matrix.os == 'ubuntu-latest'
-        run: sudo apt-get install gcc-multilib
+        run: sudo apt-get update && sudo apt-get install gcc-multilib
```

Comment:
do we need update?

Reply:
Yes -- the CI images are not regenerated frequently enough, so installing `gcc-multilib` will sometimes refer to dead packages.